src/ontology/mondo-ingest.Makefile

```diff
@@ -489,22 +487,24 @@ slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/%_ter
 	--outpath $@
 
 .PHONY: slurp-%
-slurp-%:
-	$(MAKE) slurp/$*.tsv -B
+slurp-%: slurp/%.tsv
+	echo "$@ completed".
```

Comment:
For some reason, `make` wants to see a goal body (in this case the `echo`) to run. Else, without it, `make` claims that `slurp-omim` does not exist as a goal.

Reply:
Really? Troubleshooting `make` can be frustrating. Glad you figured this out.